package.json

```diff
@@ -34,7 +34,8 @@
   ],
   "dependencies": {
     "lodash": "^4.17.10",
-    "react-docgen": "^3.0.0-rc.1"
+    "react-docgen": "^3.0.0-rc.1",
+    "react-docgen-deprecation-handler": "1.1.1"
```

Comment:
this should only be a dev dependency since it's only used for testing

Reply:
oops my bad